From 04a8638d00cce5592b9902d4a2762918a3f2cc7b Mon Sep 17 00:00:00 2001
From: Adam Butterworth <abutterworth@edx.org>
Date: Thu, 16 Apr 2020 10:37:29 -0400
Subject: [PATCH 0102/1021] Reduce min-width of unit buttons in nav (#51)

Makes space for more units before swapping the display to a dropdown.
---
 src/index.scss | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/index.scss b/src/index.scss
index a3a34f5..9566c0b 100755
--- a/src/index.scss
+++ b/src/index.scss
@@ -186,7 +186,7 @@ $primary: #1176B2;
   .sequence-navigation-tabs {
     .btn {
       flex-basis: 100%;
-      min-width: 4rem;
+      min-width: 2rem;
     }
   }
   .sequence-navigation-dropdown {
-- 
2.34.1

